lib/database_cleaner/active_record/base.rb

```diff
@@ -15,7 +15,11 @@ def self.config_file_location
 
     class Base < DatabaseCleaner::Strategy
       def self.migration_table_name
-        ::ActiveRecord::SchemaMigration.table_name
+        if Gem::Version.new("6.0.0") <= ::ActiveRecord.version
+          ::ActiveRecord::Base.connection.schema_migration.table_name
+        else
+          ::ActiveRecord::SchemaMigration.table_name
```

Comment:
this no longer works in 7.1

please merge this one instead https://github.com/DatabaseCleaner/database_cleaner-active_record/pull/82

Reply:
That's what this PR is all about. Your should use `::ActiveRecord::Base.connection.schema_migration.table_name` from 6.0.0 onwards.